docs/sources/_index.md.t

```diff
@@ -23,7 +23,7 @@ form programmable observability **pipelines** for telemetry collection,
 processing, and delivery.
 
 {{% admonition type="note" %}}
-This page focuses mainly on [Flow mode][], the Terraform-inspired variant of Grafana Agent.
+This page focuses mainly on [Flow mode](https://grafana.com/docs/agent/latest/flow/), the Terraform-inspired variant of Grafana Agent.
```

Comment:
Presently, that link will always take you to "latest" even if you are explicitly browsing other versions.

If we replace this with the suggestion, it will resolve to the correct version of documentation inferred from the current page. 
For more information, refer to  https://grafana.com/docs/writers-toolkit/write/shortcodes/#about-version-inference.

This page is also mounted in Grafana Cloud where the `<AGENT VERSION>` is always replaced with "latest".

```suggestion
This page focuses mainly on [Flow mode](https://grafana.com/docs/agent/<AGENT VERSION>/flow/), the Terraform-inspired variant of Grafana Agent.
```

Reply:
Ah I wasn't sure if the AGENT_VERSION part would work.  We should add to or update the example in https://grafana.com/docs/writers-toolkit/write/references/#choose-the-correct-link

Reply:
:tada: https://github.com/grafana/writers-toolkit/pull/345/